File: JavaScript/mayorDeTres.js
let primerNumero = parseFloat(prompt("Ingresa el primer número: "))
let segundoNumero = parseFloat(prompt("Ingresa el segundo número: "))
let tercerNumero = parseFloat(prompt("Ingresa el tercer número: "))

if ((primerNumero > segundoNumero) && (primerNumero > segundoNumero)) {
    document.write(`El primer número es el mayor ${primerNumero}`)    

}else if ((segundoNumero > primerNumero) && (segundoNumero > tercerNumero)) {
    document.write(`El segundo número es el mayor ${segundoNumero}`)
} 
else{ 
    document.write(`El tercer número es el mayor ${tercerNumero}`)
}

File: Python/mayorDeTres.py
primerNumero = float(input("Ingresa primer número: "))
segundoNumero = float(input("Ingresa segundo número: "))
tercerNumero = float(input("Ingresa tercer número: "))

if ((primerNumero > segundoNumero) and (primerNumero > tercerNumero)):
    print(f"El primer número es el mayor {primerNumero}")
elif ((segundoNumero > primerNumero) and (segundoNumero > tercerNumero)):
    print(f"El primer número es el mayor {segundoNumero}")
else:
    print(f"El primer número es el mayor {tercerNumero}")